config.json

```diff
@@ -581,6 +581,16 @@
         "difficulty": 10,
         "practices": [
         ]
+      },
```

Comment:
Lets try to order these by difficulty... IIRC iHID did say they are still presented in order somewhere... so I'd say just move this before `forth` (since we don't have difficulties pegged yet)...?

Reply:
Practice exercises are indeed listed in order.